src/backend/gslice.c

```diff
@@ -0,0 +1,256 @@
+
+/*
+ * Copyright (c) 2016 by Digital Mars
+ * All Rights Reserved
+ * written by Walter Bright
+ * http://www.digitalmars.com
+ * Distributed under the Boost Software License, Version 1.0.
+ * http://www.boost.org/LICENSE_1_0.txt
+ * https://github.com/dlang/dmd/blob/master/src/backend/gslice.c
+ */
+
+#if (SCPP || MARS) && !HTOD
+
+#include        <stdio.h>
+#include        <string.h>
+#include        <stdlib.h>
+#include        <time.h>
+
+#include        "cc.h"
+#include        "el.h"
+#include        "go.h"
+#include        "oper.h"
+#include        "global.h"
+#include        "type.h"
+#include        "code.h"
+
+static char __file__[] = __FILE__;      /* for tassert.h                */
+#include        "tassert.h"
+
```

Comment:
A brief overview of what and this module does and how it does it would be really friendly.